lablib/operators/color.py

```diff
@@ -6,34 +6,7 @@
 import PyOpenColorIO as OCIO
 
 from .base import BaseOperator
-
-
-def get_direction(direction: Union[str, int]) -> int:
-    """Get the direction for OCIO FileTransform.
-
-    Attributes:
-        direction (Union[str, int]): The direction.
-
-    Returns:
-        int: The direction.
-    """
-    if direction == "inverse":
-        return OCIO.TransformDirection.TRANSFORM_DIR_INVERSE
-    return OCIO.TransformDirection.TRANSFORM_DIR_FORWARD
-
-
-def get_interpolation(interpolation: str) -> int:
-    if interpolation == "linear":
-        return OCIO.Interpolation.INTERP_LINEAR
-    elif interpolation == "best":
-        return OCIO.Interpolation.INTERP_BEST
-    elif interpolation == "nearest":
-        return OCIO.Interpolation.INTERP_NEAREST
-    elif interpolation == "tetrahedral":
-        return OCIO.Interpolation.INTERP_TETRAHEDRAL
-    elif interpolation == "cubic":
-        return OCIO.Interpolation.INTERP_CUBIC
-    return OCIO.Interpolation.INTERP_DEFAULT
+from .utils import get_direction, get_interpolation
```

Comment:
not sure whether to start a new file for that as currently those functions are only called in a color related context right?
can't really come up with examples for other util functions but i'm fine with that

Reply:
Yeah, for now it could be part of color.py.

Reply:
okay cool. then let's add the utility functions `get_direction()` and `get_interpolation()` back to `color.py`.
once we have a usecase for more operator-related utility functions we can start the `utils.py` in `lablib/operators`